src/roadmap/release-checklist.md

```diff
@@ -56,7 +56,20 @@ be in a new directory called `workflow-v2.0.0`.
         BRANCH="release-${WORKFLOW_RELEASE}" NEW="true" make git-checkout-branch
         BRANCH="release-${WORKFLOW_RELEASE}" make git-push-branch #(can use DRY_RUN=true)
 
-# Step 2: Create New Helm Classic Charts
+# Step 2: Update Jenkins Jobs
```

Comment:
@arschles true... something worth adding in [deisrel](https://github.com/deis/deisrel) perhaps?  We can create a ticket there if it sounds appropriate.